cinema/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from cinema.views import movie_list, one_movie, movie_add, movie_update, movie_delete
+
+urlpatterns = [
+    path('movies/', movie_list, name='movie-list'),
+    path('movies/<int:pk>/', one_movie, name='one-movie'),
+    path('movies/create/', movie_add, name='add-movie'),
+    path('movies/<int:pk>/update/', movie_update, name='movie-update'),
+    path('movies/<int:pk>/delete/', movie_delete, name='movie-delete')
+]
```

Comment:
it's not REST, you should use only two paths and two views `movie_detail` and `movie_list`. Fix it)

Reply:
Done, hope this one is better)